src/BTR3.ohm

```diff
@@ -102,8 +102,10 @@ BTR3 {
     bankReferenceNumber = alphaNumeric*
     customerReferenceNumber = alphaNumeric*
     detailText  (Text relaxed for CBA BAI2 format does not contain , /  Optionally followed by /) 
-                = (~ eor any) *
+                = & "/" // null text field
+                | text * // normal text field
```

Comment:
Here, we need to do a lookahead for a `/` (in the file, this would look like `,/` because `detailText` is preceded by `delim`. In this case, the text is null, and trying to parse it as a `text` would cause the expression to be unreasonably complex